deployment/ccip/sequence/evm/v1_6/seq_fee_quoter.go

```diff
@@ -0,0 +1,40 @@
+package v1_6
+
+import (
+	"fmt"
+
+	"github.com/Masterminds/semver/v3"
+
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/v1_6_0/fee_quoter"
+	cldf "github.com/smartcontractkit/chainlink-deployments-framework/deployment"
+	"github.com/smartcontractkit/chainlink-deployments-framework/operations"
+
+	ccipops "github.com/smartcontractkit/chainlink/deployment/ccip/operation/evm/v1_6"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/shared/opsutil"
+)
+
+type FeeQuoterApplyDestChainConfigUpdatesSequenceInput struct {
+	UpdatesByChain map[uint64]opsutil.EVMCallInput[[]fee_quoter.FeeQuoterDestChainConfigArgs]
+}
+
+var (
+	FeeQuoterApplyDestChainConfigUpdatesSequence = operations.NewSequence(
+		"FeeQuoterApplyDestChainConfigUpdatesSequence",
+		semver.MustParse("1.0.0"),
+		"Apply updates to destination chain configs on the FeeQuoter 1.6.0 contract across multiple EVM chains",
+		func(b operations.Bundle, chains map[uint64]cldf.Chain, input FeeQuoterApplyDestChainConfigUpdatesSequenceInput) (map[uint64][]opsutil.EVMCallOutput, error) {
```

Comment:
```
cldf_evm "github.com/smartcontractkit/chainlink-deployments-framework/chain/evm"
```

```suggestion
		func(b operations.Bundle, chains map[uint64]cldf_evm.Chain, input FeeQuoterApplyDestChainConfigUpdatesSequenceInput) (map[uint64][]opsutil.EVMCallOutput, error) {
```

 please use `cldf_evm.Chain` going forward as we will be removing the cldf.Chain soon

Reply:
Will resolve in a follow up: https://github.com/smartcontractkit/chainlink/pull/17977